ForearmPositionGoalController:
    type: valkyrie_translator/JointPositionGoalController
    publish_est_robot_state: false
    command_channel: "DESIRED_FOREARM_ANGLES"
    command_feedback_channel: "FOREARM_COMMAND_FEEDBACK"
    control_state_channel: "FOREARM_STATE"
    control_state_publish_frequency: 80
    joints:
      - leftForearmYaw
      - leftWristRoll
      - leftWristPitch
      - rightForearmYaw
      - rightWristRoll
      - rightWristPitch
    joint_velocity_limit: 0.262 # rad/s
    joint_limits:
      leftForearmYaw:
        has_position_limits: true
        min_position: -2.019
        max_position: 3.14
      leftWristRoll:
        has_position_limits: true
        min_position: -0.62
        max_position: 0.625
      leftWristPitch:
        has_position_limits: true
        min_position: -0.36
        max_position: 0.49
      rightForearmYaw:
        has_position_limits: true
        min_position: -2.019
        max_position: 3.14
      rightWristRoll:
        has_position_limits: true
        min_position: -0.625
        max_position: 0.62
      rightWristPitch:
        has_position_limits: true
        min_position: -0.49
        max_position: 0.36
